plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -1392,6 +1392,32 @@ func TestEBSVolumeCountConflicts(t *testing.T) {
 			},
 		},
 	}
+	deletedPVCPod := &api.Pod{
```

Comment:
I was confused about the test. Both deletedPVCPod and deletedPVPod seem to be basically the same (pod with a volume with a volume source with a PVC with a name that doesn't resolve), so how are you testing both the

`c.pvcInfo.GetPersistentVolumeClaimInfo(namespace, pvcName) != nil`
and
`c.pvInfo.GetPersistentVolumeInfo(pvName) != nil`
paths?